corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -159,6 +159,14 @@ <h4 class="modal-title">
                           <input type="checkbox" data-bind="checked: hidden" />
                         </div>
                       </div>
+                      <div class="form-group">
+                        <label class="control-label col-xs-12 col-sm-3">
+                          {% trans "Exclude From Search Filters" %}
```

Comment:
A help bubble would be useful here.

Reply:
Working on this. There seems to be some weirdness between the help bubble and the modal.